tests/scenario/test_mimir_cluster_interface.py

```diff
@@ -114,9 +114,10 @@ def test_address_collection(workers_addresses):
 
 def test_requirer_getters():
     cfg = {"a": "b"}
+    lgs = {}
     relation = Relation(
         "mimir-cluster-require",
-        remote_app_data=MimirClusterProviderAppData(mimir_config=cfg).dump(),
+        remote_app_data=MimirClusterProviderAppData(mimir_config=cfg, loki_endpoints=lgs).dump(),
```

Comment:
this change should not be necessary, right?

Reply:
Right, this is a super old change I didn't even notice was there :)